docs/content/core/logger.mdx

```diff
@@ -406,3 +406,23 @@ def test_lambda_handler(lambda_handler, lambda_context):
     test_event = {'test': 'event'}
     lambda_handler(test_event, lambda_context) # this will now have a Context object populated
 ```
+
+## FAQ
+
+**Why don't I see my library logs?**
+
+Because Lambda configures the root logger for you and using the `Logger` utility would then result in duplicate log lines
+it removes the handler from the root logger.
+
+**How can I enable boto3 and botocore library logging?**
+
+You can enable the `botocore` and `boto3` logs by adding the following:
```

Comment:
Suggestion: add some context as to what this function from boto does, since not everyone might know ;)

TL;DR: 'set_stream_logger' configures boto package logger with a Log Handler (stdout) and log level as DEBUG.

Reply:
Makes sense I added the context!